pyemma/msm/api.py

```diff
@@ -296,7 +296,8 @@ def markov_model(P, dt_model='1 step'):
 def estimate_markov_model(dtrajs, lag, reversible=True, statdist=None,
                           count_mode='sliding', weights='empirical',
                           sparse=False, connectivity='largest',
-                          dt_traj='1 step', maxiter=1000000, maxerr=1e-8):
+                          dt_traj='1 step', maxiter=1000000, maxerr=1e-8,
+                          trajectory_bootstrap=True):
```

Comment:
I would not raise this parameter to the API function. It's enough to have it in the Object. In particular, the context is unclear here, because trajectory_bootstrap is only used for determining the rank of a count matrix within an OOM correction, which is ununderstandable for the majority of users that want to just estimate MSMs. So API functions should be kept relatively simple, and special parameter settings should be handled by reasonable defaults at this level. Using the objects instead of API functions allows more freedom.